Jenkinsfile

```diff
@@ -32,8 +32,16 @@ pipeline {
         BOOTSTRAP_FUNCTIONS="${CICD_URL}/src/bootstrap.sh"
         CURR_TIME=sh(script: "date +%s", returnStdout: true).trim()
     }
-
+ 
     stages {
+        stage('Did you run pre-push?') {
+            steps {
+                withVault([configuration: configuration, vaultSecrets: secrets]) {
```

Comment:
You can remove the `withVault` block here because this requires no creds right? So this stage will look more like https://github.com/RedHatInsights/clowder/pull/1034/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R55